src/main/java/com/sprint/mission/discodeit/DiscodeitApplication.java

```diff
@@ -0,0 +1,124 @@
+package com.sprint.mission.discodeit;
+
+import com.sprint.mission.discodeit.dto.entity.BinaryContent;
+import com.sprint.mission.discodeit.dto.entity.Channel;
+import com.sprint.mission.discodeit.dto.entity.ChannelGroup;
+import com.sprint.mission.discodeit.dto.entity.Message;
+import com.sprint.mission.discodeit.dto.entity.User;
+import com.sprint.mission.discodeit.dto.form.ChannelUpdateDto;
+import com.sprint.mission.discodeit.dto.form.MessageWithContents;
+import com.sprint.mission.discodeit.dto.form.PrivateChannelDto;
+import com.sprint.mission.discodeit.dto.form.PublicChannelDto;
+import com.sprint.mission.discodeit.dto.form.UserUpdateDto;
+import com.sprint.mission.discodeit.service.ChannelService;
+import com.sprint.mission.discodeit.service.MessageService;
+import com.sprint.mission.discodeit.service.UserService;
+import com.sprint.mission.discodeit.service.basic.BasicChannelService;
+import com.sprint.mission.discodeit.service.basic.BasicMessageService;
+import com.sprint.mission.discodeit.service.basic.BasicUserService;
+import com.sprint.mission.discodeit.service.basic.NetworkService;
+import com.sprint.mission.discodeit.util.JavaAppConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Scanner;
+import java.util.UUID;
+
+@Slf4j
+@SpringBootApplication
+public class DiscodeitApplication {
+	public static void main(String[] args) {
+		ApplicationContext ac=SpringApplication.run(DiscodeitApplication.class, args);
+		UserService userService=ac.getBean(BasicUserService.class);
+		ChannelService channelService=ac.getBean(BasicChannelService.class);
+		MessageService messageService=ac.getBean(BasicMessageService.class);
```

Comment:
Spring Context을 활용하여 빈을 초기화할 땐 구현체보단 인터페이스로 가져오는게 좋을 것 같아요.  
```suggestion
		UserService userService=ac.getBean(UserService.class);
		ChannelService channelService=ac.getBean(ChannelService.class);
		MessageService messageService=ac.getBean(MessageService.class);
```